src/UI/class-recommended-widget.php

```diff
@@ -143,7 +143,7 @@ public function widget( $args, $widget_settings ): void /* @phpstan-ignore-line
 		// Set up the variables.
 		$api_url = $this->get_api_url(
 			$this->parsely->get_site_id(),
-			$instance['published_within'],
+			(int) $instance['published_within'], // @phpstan-ignore-line
```

Comment:
Quick fix for a fatal in the Recommended Widget, which is to be dropped in a future release.